config.dev.py

```diff
@@ -1 +1,3 @@
 SAM_ROOT = "https://sam.dsde-dev.broadinstitute.org"
```

Comment:
These `config.*.py` files will probably migrate to terra-helm/terra-helmfile somehow